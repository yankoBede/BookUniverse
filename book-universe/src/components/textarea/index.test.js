import React from 'react'
import renderer from 'react-test-renderer'
import TextArea from './index'

describe('Text Area Component', () => {
    it('should render title with passes text', () => {
        const tree = renderer.create(
            <TextArea label="test text area label"
            id="test-text-id"
            name="test-text-name"
            value="test-text-value"
            onChange={() => { console.log('text cnahged')}}
            divClass="div-class" 
            inputClass="input-class"
            />
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})