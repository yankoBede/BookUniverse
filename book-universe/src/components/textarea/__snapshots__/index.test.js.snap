// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Area Component should render title with passes text 1`] = `
<div
  className="div-class"
>
  <label
    htmlFor="test-text-id"
  >
    test text area label
  </label>
  <textarea
    className="input-class"
    id="test-text-id"
    name="test-text-name"
    onChange={[Function]}
    rows="10"
    value="test-text-value"
  />
</div>
`;
